bucket_99/rofi/specification

```diff
@@ -0,0 +1,48 @@
+DEF[PORTVERSION]=	1.5.1
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		rofi
+VERSION=		${PORTVERSION}
+KEYWORDS=		x11
+VARIANTS=		standard
+SDESC[standard]=	Window switcher, run dialog and dmenu replacement
+HOMEPAGE=		https://github.com/DaveDavenport/rofi
+CONTACT=		nobody
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		https://github.com/DaveDavenport/rofi/releases/download/${PORTVERSION}/
+DISTFILE[1]=		rofi-${PORTVERSION}.tar.gz:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		MIT:single
+LICENSE_FILE=		MIT:{{WRKSRC}}/COPYING
+LICENSE_TERMS=		single:{{WRKSRC}}/COPYING
```

Comment:
Remove LICENSE_TERMS, the MIT license stands alone.  LIcenses like GPL and APACHE20 require terms.